### DISCUSSION TITLE
Mexico_City tzdata update

### DISCUSSION THREAD

**Discussion by alfredo-rgzm** [f1multiviewer/issue-tracker#515] [General]
Last year the Mexico_City timezone stopped using daylight saving time. For some reazon, this time of the year, Multiviewer app is always off by 1 hour plus or minus depending where the races happens. Not sure how the app gets its timezone many apps in my computers show up properly now but Multiviewer App does not seem to calculate locally the times hence the offset I think.
In theory all you need is to keep up to date your tzdata updated.

**Comment by LapsTimeOFF** [comment]
Hey, we have updated the framework that MultiViewer uses, is this issue still occuring? If so please open an issue in the issues tab.
